css/style.css

```diff
@@ -114,6 +114,7 @@ blockquote em {
 }
 
 header {
+  padding-top: 100px;
```

Comment:
I thought this would be a problem if/when the app bundles the code, but there doesn't actually seem to be a build step for the css. So this is ok.